src/controller/servicemgr/executor/containerexecutor/ce_docker.go

```diff
@@ -22,13 +22,12 @@ import (
 	"io"
 	"os"
 
-	"docker.io/go-docker"
```

Comment:
@tdrozdovsky As @t25kim has reported as issue #104, it triggers the build failure, and it can be resolved by PR #105 suggested by @t25kim. @t25kim Thank you for the kind evaluation and a timely suggestion.

@suresh-lc We need to setup the CI/CD system as soon as possible integrating with the current PR process. What do you think?

Reply:
@MoonkiHong , Brett had previously mentioned about CI/CD system supported by LF Edge as such. He had asked to raise a ticket for the same to be set up for Home Edge also. So let me discuss with Brett on the same and get back.